features/step_definitions/assertion_steps.rb

```diff
@@ -299,12 +301,12 @@ def current_path_without_locale(path)
 end
 
 
-# Tests that an input or button with the given label is disabled.
-Then /^the "([^\"]*)" field should( not)? be set to "([^\"]*)"$/ do |label, negate, text_value|
+# Tests that an input has a given value
+Then /^the t\("([^"]*)"\) field should be set to "([^\"]*)"$/ do |i18n_key, text_value|
```

Comment:
You're right. The step in 425 also works. But due to the first_name and last_name being nested attributes it would require to use "membership_application_user_attributes_last_name" as fieldname in the step. All in all, I prefer to leave it as is.